src/Utils/String.ts

```diff
@@ -0,0 +1,13 @@
+const quotes = [ "'", '"' ];
+
+export function strip(input: string, chars: string[] = quotes): string {
```

Comment:
I don't think stripping quotes is enough in all cases. For example, what if the string has quoted characters? Like `"Hello\"World"`, which should be `Hello"World` and not `Hello"World`. Do you know whether Typescript has a way to unqote strings in a more robust way?

Reply:
Looks like TypeScript does have a `stripQuotes` utility: https://github.com/microsoft/TypeScript/blob/387c917765793773a6f7184bab84e6f5956f44fc/src/compiler/utilities.ts#L3158-L3164 which essentially does the same as what I implemented, albeit more efficient. Not sure what your example is illustrating both the outputs look the same?

Reply:
I guess it's fine not to handle escaping then. Let's use import the typescript util so we don't re-invent the wheel here.

Thank for finding this!